addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -295,7 +303,31 @@ func _ready():
 			_movable.last_scale = scale
 			_movable.update_terrain()
 
+func connect_trigger_events():
+	if not is_trigger:
+		return
```

Comment:
Is this check necessary since this method only appears to be called once inside an if-statement that already checks `is_trigger`?

Reply:
Well we could also use `assert` here I assume. I put that check as a security